cypress/support/cypress-commands/assertion.js

```diff
@@ -618,6 +618,9 @@ Cypress.Commands.add(
 Cypress.Commands.add(
   'saveEventResponse',
   (response, methodOrEventObject, eventName, eventExpected, isNullCase) => {
+    fireLog.info(
+      `Received response for the event ${eventName} is ${response.eventResponse ? JSON.stringify(response.eventResponse) : JSON.stringify(response)}`
+    );
     const eventNameForLog = eventName.split('-')[0];
```

Comment:
can we revert this log? we can include that in one of the other tickets. I want this to be tested properly

Reply:
reverted log changes

Reply:
logs are reverted hence resolving the comment